boards/px4/fmu-v5/src/can.c

```diff
@@ -1,129 +0,0 @@
-/****************************************************************************
```

Comment:
I have had request to use the nuttx can driver. It is here to support those uses.